infoschema/metric_schema.go

```diff
@@ -925,4 +925,1055 @@ var MetricTableMap = map[string]MetricTableDef{
+		Labels:  []string{"instance"},
+		Comment: "The average rate of writing bytes to Regions per TiKV instance",
+	},
+	"tikv_region_written_bytes": {
+		PromQL: `sum(rate(tikv_region_written_bytes_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_region_average_written_keys": {
+		PromQL:  `sum(rate(tikv_region_written_keys_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance) / sum(rate(tikv_region_written_keys_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The average rate of written keys to Regions per TiKV instance",
+	},
+	"tikv_region_written_keys": {
+		PromQL: `sum(rate(tikv_region_written_keys_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_request_batch_avg": {
+		PromQL:  `sum(rate(tikv_server_request_batch_ratio_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance) / sum(rate(tikv_server_request_batch_ratio_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The ratio of request batch output to input per TiKV instance",
+	},
+	"tikv_request_batch_ratio": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_server_request_batch_ratio_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,type,instance))`,
+		Labels:   []string{"instance", "type"},
+		Quantile: 0.99,
+		Comment:  "The ratio of request batch output to input per TiKV instance",
+	},
+	"tikv_request_batch_size_avg": {
+		PromQL:  `sum(rate(tikv_server_request_batch_size_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance) / sum(rate(tikv_server_request_batch_size_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The avg size of requests into request batch per TiKV instance",
+	},
+	"tikv_request_batch_size": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_server_request_batch_size_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,type,instance))`,
+		Labels:   []string{"instance", "type"},
+		Quantile: 0.99,
+		Comment:  "The size of requests into request batch per TiKV instance",
+	},
+
+	"tikv_grpc_messge_duration": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_grpc_msg_duration_seconds_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,type,instance))`,
+		Labels:   []string{"instance", "type"},
+		Quantile: 0.99,
+		Comment:  "The execution time of gRPC message",
+	},
+	"tikv_average_grpc_messge_duration": {
+		PromQL: `sum(rate(tikv_grpc_msg_duration_seconds_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance) / sum(rate(tikv_grpc_msg_duration_seconds_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance)`,
+		Labels: []string{"instance", "type"},
+	},
```

Comment:
Some items missed comments, is it expected?

Reply:
Since those definitions are extracted from `tidb-ansible/scripts`, If `tidb-ansible/scripts` doesn't have any comment, so here will not have comment.

I think the name is meaningful.